odd-platform-ui/src/components/DataEntityDetails/DatasetStructure/DatasetStructureTable/DatasetStructureList/DatasetStructureItem/DatasetFieldInfoEditForm/LabelsAutocomplete/LabelsAutocomplete.tsx

```diff
@@ -18,9 +14,7 @@ import { UseFieldArrayReturn } from 'react-hook-form';
 type FilterOption = Omit<Label, 'id'> & Partial<Label>;
 
 interface LabelsAutocompleteProps {
-  searchLabels: (
-    params: LabelApiGetLabelListRequest
-  ) => Promise<LabelsResponse>;
+  searchLabels: (params: LabelApiGetLabelListRequest) => Promise<any>;
```

Comment:
In this case, when you are wrote a new thunk with `createAsyncThunk`, you don't need to pass this thunks in LabelsAutocompleteContainer via `mapDispatchToProps`. 
You can remove this prop and just import `fetchLabelsList` func in the LabelsAutocomplete component and dispatch it when you need. 
In this way you will avoid this `any` typing.